src/cargo/ops/cargo_new.rs

```diff
@@ -487,10 +487,14 @@ impl IgnoreList {
             _ => &self.ignore,
         };
 
-        let mut out = "\n\n#Added by cargo\n\
-                       #\n\
-                       #already existing elements are commented out\n\n"
-            .to_string();
+        let mut out = "\n\n#Added by cargo\n".to_string();
```

Comment:
I'm tempted to add some spaces before the `#` just for _subjectively nicer_ formatting.

```
/foo
/target

# Added by cargo
#
# already existing elements are commented out

# /target
**/*.rs.bk
```

Thoughts?